fineract-provider/src/main/java/org/apache/fineract/portfolio/savings/domain/SavingsAccountTransaction.java

```diff
@@ -303,9 +304,11 @@ public static void updateTaxDetails(final Map<TaxComponent, BigDecimal> taxDetai
     }
 
     public static SavingsAccountTransaction copyTransaction(SavingsAccountTransaction accountTransaction) {
+        LocalDateTime date = DateUtils.getLocalDateTimeOfTenant();
+        Date createdDate = Date.from(date.atZone(ZoneId.systemDefault()).toInstant());
```

Comment:
Why all this instead of just doing a DateUtils.getLocalDate()? It's much simpler and gets the current date properly.